src/parameters/update_parameters.jl

```diff
@@ -331,3 +331,46 @@ _has_variable_cost_parameter(component::PSY.Component) =
     _has_variable_cost_parameter(PSY.get_operation_cost(component))
 _has_variable_cost_parameter(::PSY.MarketBidCost) = true
 _has_variable_cost_parameter(::T) where {T <: PSY.OperationalCost} = false
+
+function _update_pwl_cost_expression(
```

Comment:
@sourabhdalvi confirms this is consistent with the changes to PWL in `objective_function.jl`